src/apurva/Mall.java

```diff
@@ -0,0 +1,50 @@
+package apurva;
+
+class Mall {
+	void itemAddedToCart(int items) {
+		System.out.println(items);
```

Comment:
why we are printing statement inside method, this are method is not for display items,